controllers/common/common.go

```diff
@@ -245,6 +247,68 @@ func FullNameForPolicy(plc *policiesv1.Policy) string {
 	return plc.GetNamespace() + "." + plc.GetName()
 }
 
+// GetRepPoliciesInPlacementBinding returns a list of the replicated policies that are either direct subjects of
+// the given PlacementBinding, or are in PolicySets that are subjects of the PlacementBinding.
+// The list items are guaranteed to be unique (for example if a policy is in multiple sets).
+func GetRepPoliciesInPlacementBinding(
+	ctx context.Context, c client.Client, pb *policiesv1.PlacementBinding,
+) []reconcile.Request {
+	decisions, err := GetDecisions(ctx, c, pb)
+	if err != nil {
+		return []reconcile.Request{}
+	}
+	// Use this for removing duplicated policies
+	table := map[reconcile.Request]bool{}
+
+	for _, subject := range pb.Subjects {
```

Comment:
Instead of duplicating this code, could you please call `GetPoliciesInPlacementBinding` and then use that result and `decisions` to generate the reconcile requests for all replicated policies?

Reply:
I do think that it'd be nice if `GetPoliciesInPlacementBinding` ensured uniqueness, though the queue used by controller-runtime will consolidate them nearly 100% of the time.